### DISCUSSION TITLE
Relicense LAMMPS module to GPLv2 license

### DISCUSSION THREAD

**Discussion by njzjz** [deepmodeling/deepmd-kit#3637] [Show and tell]
Hi developers,

We plan to contribute the DeePMD-kit LAMMPS module (USER-DEEPMD) to the [LAMMPS source code](https://github.com/lammps/lammps). LAMMPS is licensed under GPLv2-only license, but DeePMD-kit is licensed under LGPLv3. As they are incompatible, we ask all contributors to the USER-DEEPMD module to comment here to agree to relicense the previous contribution to GPLv2-only. **If you disagree, we may remove your contribution when contributing to LAMMPS.**

The source code that needs to be relicensed: `pair_deepmd.{cpp,h}`, `pppm_dplr.{cpp,h}`, `fix_dplr.{cpp,h}`, `compute_deeptensor_atom.{cpp,h}`, `fix_ttm_dp.h`.

Contributors to these files: @wanghan-iapcm, @denghuilu, @njzjz, @y1xiaoc, @mingzhong15, @Yi-FanLi, @hztttt, @shiruosong, @Cloudac7, @ChiahsinChu, @link89, @DavideTisi

**Comment by DavideTisi** [comment]
I agree to relicense the contribution

**Comment by mingzhong15** [comment]
I agree to relicense the contribution


---- Replied Message ----
| From | Jinzhe ***@***.***> |
| Date | 4/2/2024 16:37 |
| To | ***@***.***> |
| Cc | ***@***.***> ,
***@***.***> |
| Subject | [deepmodeling/deepmd-kit] Relicense LAMMPS module to GPL-2.0 license (Discussion #3637) |

Hi developers,

We plan to contribute the DeePMD-kit LAMMPS module (USER-DEEPMD) to the LAMMPS source code. LAMMPS is licensed under GPL-2.0 license, but DeePMD-kit is licensed under LGPL-3.0. As they are incompatible, we ask all contributors to the USER-DEEPMD module to comment here to agree to relicense the previous contribution to GPL-2.0. If you disagree, we may remove your contribution when contributing to LAMMPS.

The source code that needs to be relicensed: pair_deepmd.{cpp,h}, pppm_dplr.{cpp,h}, fix_dplr.{cpp,h}, compute_deeptensor_atom.{cpp,h}, fix_ttm_dp.h.

Contributors to these files: @wanghan-iapcm, @denghuilu, @njzjz, @y1xiaoc, @mingzhong15, @Yi-FanLi, @hztttt, @shiruosong, @Cloudac7, @ChiahsinChu, @link89, @DavideTisi

**Comment by shiruosong** [comment]
I agree to relicense the contribution

**Comment by link89** [comment]
I agree to relicense the contribution

**Comment by Cloudac7** [comment]
I agree to relicense the contribution

**Comment by y1xiaoc** [comment]
I agree to relicense the contribution

**Comment by wanghan-iapcm** [comment]
I agree to relicense the contribution

**Comment by denghuilu** [comment]
I agree to relicense the contribution

**Comment by ChiahsinChu** [comment]
I agree to relicense the contribution.

**Comment by Yi-FanLi** [comment]
I agree to relicense the contribution
